projects/demo-maps/src/app/route-components/route-example-custom-layers/sunlight/sunlight.component.html

```diff
@@ -0,0 +1,12 @@
+<form clrForm [formGroup]="sunlightForm">
```

Comment:
If this is a normal UI component then it should be in the components folder.

Reply:
Done!